src/drivers/imu/invensense/iam20680hp/InvenSense_IAM20680HP_registers.hpp

```diff
@@ -0,0 +1,193 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2020-2021 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
 *   Copyright (c) 2023 PX4 Development Team. All rights reserved.
```

I set the year to when it's first added.